### PR TITLE
Feature: paid concurrency

### PR DESCRIPTION
**Pull request by artemii-yanushevskyi**

### CODE REVIEW

File: content/billing/pricing.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ Usage on macOS standard VM that exceeds 500 minutes is charged at rate shown bel
 
 Builds on macOS premium VM, Linux standard VM, and Linux premium VM do not have free build minutes and are charged at the rates shown below. 
 
+Each extra build concurrency allows to run an additional build in paralel. For example, two extra build concurrencies allow to run three builds in paralel. Consider Business plan or Enterprise plan if more than three concurrent builds are required.
```
Comment (helinanever) [inline content/billing/pricing.md:44]: The build concurrency information should not be under Pricing for personal accounts but under Pricing for teams. 
Also need to update line 10 and say that additional concurrency is available for teams on the pay as you go plan.

**Review by helinanever — CHANGES_REQUESTED** [review]

File: content/billing/pricing.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ weight: 2
 
 Pay as you go pricing allows you to pay for only what you use. 
 
-Currently it is only possible to run one build at a time with pay as you go.
+Extra build concurrencies allow Teams to run up to three paralel builds.
```
Comment (helinanever) [inline content/billing/pricing.md:10]:
```suggestion
Extra build concurrencies allow Teams to run up to three parallel builds.
```

**Review by craastad — APPROVED** [review]

File: content/billing/pricing.md [hunk 3]
```diff
@@ -54,13 +54,16 @@ For teams, all build minutes using macOS standard VM, macOS premium VM, Linux st
 
 Team users that were active in the current billing period are charged $10/month. To see how users are counted please see [here](https://docs.codemagic.io/teams/users/)
 
+Each extra build concurrency allows to run an additional build in paralel. For example, two extra build concurrencies allow to run three builds in paralel. Consider Business plan or Enterprise plan if more than three concurrent builds are required.
```
Comment (helinanever) [inline content/billing/pricing.md:57]:
```suggestion
Each extra build concurrency allows running an additional build in parallel. For example, two extra build concurrencies allow running three builds in parallel. Consider Business plan or Enterprise plan if more than three concurrent builds are required.
```

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**